appveyor/scripts/setBuildVersionVars.ps1

```diff
@@ -1,5 +1,13 @@
 $ErrorActionPreference = "Stop";
-# iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+
+# Accessing Windows build worker via Remote Desktop (RDP)
+# Set a password via Appveyor settings to enable, ensure passsword requirements are met

```

Comment:
It would be good to add a suggestion to remove the password when done

```suggestion
# Accessing Windows build worker via Remote Desktop (RDP).
# Set a password via AppVeyor settings to enable, ensure password requirements are met.
# Remove password via AppVeyor settings once RDP access is done.
```

Reply:
Yep, agreed.